noir/noir-repo/tooling/fuzzer/src/dictionary/mod.rs

```diff
@@ -84,7 +84,10 @@ fn build_dictionary_from_circuit<F: AcirField>(circuit: &Circuit<F>) -> HashSet<
             }
 
             Opcode::BlackBoxFuncCall(BlackBoxFuncCall::RANGE {
-                input: FunctionInput { num_bits, .. },
+                input: FunctionInput::Constant(ConstantInput { num_bits, .. }),
```

Comment:
We should write the constant as well into the dictionary here

Reply:
Done